reference/checkouts.v3.yml

```diff
@@ -2363,3 +2363,9 @@ responses:
           order_confirmation_use_custom_checkout_script: false
           custom_order_confirmation_script_url: 'https://example.com/custom-order-confirmation-script.js'
         meta: {}
+      'webdav: prefix':
```

Comment:
```suggestion
      'webdav protocol':
```

Reply:
`webdav:` is a protocol scheme -- [read more](https://en.wikipedia.org/wiki/WebDAV) if you wish

Reply:
I'm not sure where the value of this YAML key shows up or what is signifies on the rendered page, so my suggestion might be in the wrong format.

Reply:
That's correct Wevdav is a protocol. Also FYI we haven't updated the code yet to allow webdav to be used. This will be implemented likely next sprint as part of https://jira.bigcommerce.com/browse/CHECKOUT-6576

Reply:
@valentindellangela Thank you for this feedback.  I accepted the changes and will wait to merge this PR. Please let me know when it is in the code and ready for use.